Sources/ACKategories/SwiftUIExtensions/View+FrameSize.swift

```diff
@@ -0,0 +1,17 @@
+import SwiftUI
+
+@available(iOS 13, *)
```

Comment:
The availability should be fixed for other platforms, probably check other extensions that are SwiftUI specific. That should fix the CI build.